docs/config/README.md

```diff
@@ -350,6 +351,29 @@ modules you explicitly add to the format will not be duplicated. Eg.
 format = '$all$directory$character'
 ```
 
+### Profiles
+
+Profiles can be used to set a key-value-map with alternate versions of the top-level `format` (the main prompt format string) and `right_format` (the same for the `right_prompt`). To quickly switch to a different profile of the current prompt, you can set the `STARSHIP_PROFILE` environment variable to the name of the profile, for instance by running `export STARSHIP_PROMPT=my_profile` or `$ENV:STARSHIP_PROMPT="my_profile"` in PowerShell. Alternatively, it can also be set via an `--profile=my_profile` flag for commands like `starship prompt` and `starship explain`.
```

Comment:
If I read `src/config.rs` correctly, the variable is `STARSHIP_PROFILE` and **not** `STARSHIP_PROMPT`. Is this true, or did I miss something?

Reply:
Thanks, I've fixed this.